src/resources/pages.json

```diff
@@ -12,6 +12,9 @@
   "appletOverview": "/dashboard/:appletId/overview",
   "appletActivities": "/dashboard/:appletId/activities",
   "appletRespondents": "/dashboard/:appletId/respondents",
+  "appletRespondentDetails": "/dashboard/:appletId/respondents/:respondentId",
```

Comment:
Since these are all new routes, and @sultanofcardio's PR introduces `"/dashboard/:appletId/respondents"` (https://github.com/ChildMindInstitute/mindlogger-admin/pull/1580), I think it wouldn't hurt to update this to `"appletParticipantDetails": "/dashboard/:appletId/participants/:participantId"`, same with the other two routes

Thanks @farmerpaul for calling this out!